packages/genesys-spark-components/src/components/beta/gux-popover-beta/gux-popover.scss

```diff
@@ -0,0 +1,71 @@
+/* This is necessary to override the position of the popover on the page. By default, an HTML popover will
+   render in the middle of the screen, which we do not want */
+:popover-open {
+  position: absolute;
+  inset: unset;
+}
+
+.gux-popover-wrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: var(
+    --gse-semantic-zIndex-popover
+  ); // TODO: this will not work because the popover will display in the top layer. Figure out a solution.
+
+  min-width: 280px; // TODO: add new token once UX has added it: GDS-2672
+  padding: var(--gse-ui-popover-gap);
+  overflow: hidden;
+  background-color: var(--gse-ui-popover-backgroundColor);
+  border: none;
+  border-radius: var(--gse-ui-popover-borderRadius);
+  box-shadow: var(
+      --gse-ui-popover-boxShadow-x,
+      --gse-ui-popover-boxShadow-offsetX
+    )
+    var(--gse-ui-popover-boxShadow-y, --gse-ui-popover-boxShadow-offsetY)
+    var(--gse-ui-popover-boxShadow-blur) var(--gse-ui-popover-boxShadow-spread)
+    var(--gse-ui-tooltip-boxShadow-color);
+
+  .gux-arrow {
+    position: absolute;
+    width: var(--gse-ui-popover-anchor-width);
+    height: var(--gse-ui-popover-anchor-height);
+
+    /* 4px padding leaves some space for the shadow to bleed into */
+    padding-bottom: 4px;
+    overflow: hidden;
+  }
+
+  .gux-arrow-caret {
```

Comment:
Is the team ok if we do not display a caret anymore or is it necessary?

Reply:
it is necessary that we match the spark design https://spark.genesys.com/latest/components/feedback/popovers/usage-wA2Lqhzx